templates/layout.html

```diff
@@ -41,12 +29,14 @@
   </main>
 
   <footer data-region="footer">
-    <p class="h-card">
-      Copyright &copy; {{ config.copyright }}
-      <a href="/" class="p-name u-url">OddBird</a>
-      |
-      Established in 2008 by three odd siblings.
-      <a href="/conduct/">Code of Conduct</a>
-    </p>
+    {% block footer %}
```

Comment:
Looks like we're not using this `block` anywhere (that is, not overwriting it), but doesn't hurt to have it anyway.